### DISCUSSION TITLE
Welcome to armada-alliance Discussions!

### DISCUSSION THREAD

**Discussion by sublayerio** [armada-alliance/armada-alliance#174] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AstroWa3l** [comment]
Hi, I am Wael the co-owner of PIADA stake pool. I am an entrepreneur in the blockchain space and I hope to work in blockchain the rest of my life to help ensure that it never becomes centralized like everything else in this world. 

What do I do within this community?

Well, other than founding the alliance, I mainly work on future projects for the alliance like the DAO, tutorial creation, content creation, help with website development (as much as I am able to), marketing strategies, and much more!! 

I am looking forward to the future of the Armada Alliance and will be working on some more pretty cool projects.
